concourse/scripts/kibana_set_utc.rb

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env ruby
+
+require 'net/http'
+require 'json'
+
+def ingest_to_es(document, uri)
```

Comment:
Could we rename this to something like `update_es_index` or `upsert_es_index`?

I found `ingest` confusing because it already has a meaning in the logsearch release and our manifests.